services/apps/alcs/src/alcs/application/application-timeline/application-timeline.service.ts

```diff
@@ -1,6 +1,7 @@
 import { Injectable } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
 import { Repository } from 'typeorm';
+import { ApplicationDecisionMeetingService } from '../../application-decision/application-decision-v1/application-decision-meeting/application-decision-meeting.service';
```

Comment:
Is it correct that you are using decision version 1 ?

Reply:
I think this meeting service is only in V1 but will be used even once we move to Decision V2.